### PR TITLE
Add project roadmap

### PR DESCRIPTION
**Pull request by dlipovetsky**
/cc @justinsb @neolit123

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+The project's major goals, ranked by importance:
+
+1. Define API types.
+2. Read configuration from a file.
+3. Integrate with the [Cluster API project](https://cluster-api.sigs.k8s.io/).
+	- Support the "external etcd" use case.
+	- Work together with [kubeadm](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/), which the Cluster API project uses in its default implementation.
+4. Implement backup/restore functionality.
+	- Given an example of how to take periodic backups automatically using etcdadm.
```
Comment (neolit123) [inline ROADMAP.md]: Given / give?

Comment (dlipovetsky) [reply ROADMAP.md]: Fixed, thank you!

**Comment by neolit123** [comment]
/lgtm
/hold
feel free to hold cancel.

**Comment by neolit123** [comment]
@dlipovetsky 

> time="2019-10-28T16:25:41Z" level=fatal msg="[initsystem] Error detecting the init system: systemd not detected; ensure that `systemctl` is in the PATH"

if forgot that the test job requires systemd.
change these lines:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/etcdadm/etcdadm-presubmits.yaml#L23-L24

to:
```
        command:
        - "runner.sh"
        - "./test/e2e/init.sh"
```

**Comment by dlipovetsky** [comment]
Looks like `systemctl` isn't in the PATH. I'm trying to test locally using `pj-on-kind.sh`, but that's failing for a different reason: prow is pulling master that's behind by one commit, the one that makes the e2e script executable. Not sure why that's happening:

```
    Environment:       
      GOPROXY:             https://proxy.golang.org
      ARTIFACTS:           /logs/artifacts                                                                         
      BUILD_ID:            1188942349732941824
      BUILD_NUMBER:        1188942349732941824
      GOPATH:              /home/prow/go
      JOB_NAME:            pull-etcdadm-test-init       
      JOB_SPEC:            {"type":"presubmit","job":"pull-etcdadm-test-init","buildid":"1188942349732941824","prowjobid":"6d7c0d15-f9d0-11e9-a3fd-0242ac110006","refs":{"org":"kubernetes-sigs","repo":"etcdadm","base_ref":"master","
base_sha":"a915350fd2d652a7a0638102859145ec3e6215e1","pulls":[{"number":137,"author":"dlipovetsky","sha":"8474311087c10ccf0e2411f58650823fbfceedf7"}],"path_alias":"sigs.k8s.io/etcdadm"}}
      JOB_TYPE:            presubmit
      PROW_JOB_ID:         6d7c0d15-f9d0-11e9-a3fd-0242ac110006              
      PULL_BASE_REF:       master
      PULL_BASE_SHA:       a915350fd2d652a7a0638102859145ec3e6215e1
      PULL_NUMBER:         137
      PULL_PULL_SHA:       8474311087c10ccf0e2411f58650823fbfceedf7           
      PULL_REFS:           master:a915350fd2d652a7a0638102859145ec3e6215e1,137:8474311087c10ccf0e2411f58650823fbfceedf7
      REPO_NAME:           etcdadm
      REPO_OWNER:          kubernetes-sigs
      ENTRYPOINT_OPTIONS:  {"timeout":7200000000000,"grace_period":15000000000,"artifact_dir":"/logs/artifacts","args":["runner.sh","./test/e2e/init.sh"],"process_log":"/logs/process-log.txt","marker_file":"/logs/marker-file.txt","
metadata_file":"/logs/artifacts/metadata.json"} 
```

**Comment by neolit123** [comment]
commented on slack:
https://kubernetes.slack.com/archives/CDECRSC5U/p1572301198129800?thread_ts=1572291436.122700&cid=CDECRSC5U

> prow is pulling master that's behind by one commit

this is odd and is worthy of a #prow report?

**Comment by dlipovetsky** [comment]
> > prow is pulling master that's behind by one commit
> 
> this is odd and is worthy of a #prow report?

Thanks, I brought it up on slack: https://kubernetes.slack.com/archives/CDECRSC5U/p1572310280137700

**Pull request merged**